### ISSUE TITLE
Micro Stutters When Opened With CS:GO

### ISSUE THREAD

**Issue by ghost** [libvibrant/vibrantLinux#49]
When I leave it open and start CS:GO it micro stutters my -vulkan running CS:GO.
The solution for me was to just adjust global settings and then with my preferred vibrancy, close the application and then run CS:GO.

The micro stutters are in the rate of 1-2 seconds. Presumably because the program checks itself or checks the vibrancy, causing simple small stutter.

It's not my CPU, because it's always set to performance mode and 4.8 GHz. When I close Vibrantlinux, it stops stuttering. 

I would want to leave it open to run in the background and not mess with closing and reopening the program. It would be great also to not even run the Gui, but to just implement vibrancy in the console, because I'm running CS:GO with script anyway.

If there is a solution for me to just call my preferred vibrancy and put it in my script with CS:GO launching?
I checked -help flag and also with -h and --help, and there is nothing there. I also tried to seek for man for it, but nothing there.

Furthermore, I would appreciate greatly added console feature if it's possible, because this program is great! Thanks!

#### The bug can be easily reproduced by opening VibrantLinux, and after that without shutting it down, running CS:GO and going to any map without bots and strafing A D and seeing hard stutter with and interval of 1-2 seconds.

**Comment by Gabisonfire** [comment]
Experiencing this as well, but it seems to be very recent.

**Comment by Scrumplex** [comment]
Maybe you can try to pinpoint the frequency using a tool like MangoHud. 
vibrantLinux checks for your current window focus every second. So if the stutters occur every second, then it's indeed an issue here.

Though I am not quite sure if there is a better way for checking for window changes other than using this interval

**Comment by Gabisonfire** [comment]
Oh it's definitely vibrantlinux, its stops as soon as I turn it off. Not a big deal at the moment since I just put the setting and turn it off, but I used to leave it running, which was convenient.

**Comment by niereklamuj** [comment]
Yes, when I put my desired vibrancy and then close the program, CS:GO stops stuttering. From what I remember, it was prevailing on Vulkan and on OpenGL as well. The solution would be to just make compatible console command, not to use it with GUI. This would probably not change much in code, but it would add another feature. It would be perfect for CS:GO and other games which uses script anyway which can be edited.

**Comment by guihkx** [comment]
Are you guys using GNOME + NVIDIA by any chance?

**Comment by niereklamuj** [comment]
I'm using AMD and XFCE4. How about others?

**Comment by guihkx** [comment]
Hmm, could be a different bug than the one I'm thinking about then... But if you are able to reproduce this stutter easily, try this just to confirm my suspicion: Open a terminal window and begin monitoring the system journal by running:

    journalctl -f

And then reproduce the stutter for a few seconds. Go back to the terminal window and check if you get a lot of output there. If you don't get anything, then it's a different issue.

**Comment by niereklamuj** [comment]
Cannot reproduce stutters anymore. Maybe because installed amd amf with amdgpu pro.

**Comment by Gabisonfire** [comment]
> Are you guys using GNOME + NVIDIA by any chance?

Yes, Cinnamon more precisely. I did try journactl but nothing relevant came out.

**Comment by guihkx** [comment]
>Yes, Cinnamon more precisely. I did try journactl but nothing relevant came out.

Might be some other issue then, unfortunately. Just to be sure, instead of running `journalctl -f`, try:

```
tail -f ~/.local/share/xorg/Xorg.0.log
```

And see if stuff keeps popping up with Vibrant Linux running.
